index.html

```diff
@@ -0,0 +1,344 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <!-- bootstrap css  -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+    <!-- google fonts css  -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Chivo:ital,wght@1,700&display=swap" rel="stylesheet">
+    <!-- bootstrap cdn  -->
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.10.2/font/bootstrap-icons.css">
+    <!-- custom css  -->
+    <link rel="stylesheet" href="scss/style.css">
+</head>
+<body>
+    <!-- banner starts here div1 -->
+    <div class="banner">
+      <div class="innerdiv">
+        <div class="glow-div"></div>
+        <h2 class="banner-head-left">ethical</h2>
+        <h2 class="banner-head-right">fashion</h2>
+        <img src="image/elegant-female-model-sitting-armchair-jacket-suit-new-feminity-concept5.jpg" class="banner-img">
+        <h2 class="banner-head-left-over">ethical</h2>
+        <h2 class="banner-head-right-over">fashion</h2>
+      </div>
+     
+        <div class="nav-bar">
+          <div class="menu-wrap" id="menu-wrap">
+            <div class="container-md">
+            <div class="hamburger" id="hamburger"><div class="ham"></div></div>
+            <a href="#" class="shopingbtn"><i class="bi bi-bag"></i></a>
+            <div class="searchbar">
+              <input type="search" id="searchbox" placeholder="Search">
+              <div class="icon"></div>
+            </div>
+            <div class="menu">
+              <div>
+                <ul>
+                  <li><a href="#">Home</a></li>
+                  <li><a href="#">About</a></li>
+                  <li><a href="#">Shop</a></li>
+                  <li><a href="#">Contact</a></li>
+                </ul>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="container-md">
+        <div class="banner-div">
+          <p class="banner-para">
+            Conscious style and design production 
+            that doesn't harm working conditions,
+            the environment, and animals.
+          </p>
+          <button class="btn" type="button">SHOP NOW</div>
+      </div>
+        
+        
+        <div class="banner-flower-div">
+          <img src="image/flower.png" class="banner-flower">
+          <h2 class="flowernum">01</h2>
+        </div>
+        
+    </div>
+    <!-- banner ends here  -->
+    <!-- shopping starts here div2 -->
+    <div class="shopping">
+      <img src="image/brush.png" class="l-brush">
```

Comment:
add alt tag in all images and pass related text in it

Reply:
added alt text